quickstarts/microsoft.cache/redis-enterprise-vectordb/main.bicep

```diff
@@ -0,0 +1,44 @@
+@description('Location of all resources')
+param location string = resourceGroup().location
+
+@description('Name of the Redis Enterprise Cache')
+param redisCacheName string = ''
+
+@description('SKU of the Redis Enterprise Cache')
+param redisCacheSKU string = 'Enterprise_E10'
+
+@description('Capacity of the Redis Enterprise Cache')
+param redisCacheCapacity int
```

Comment:
need a parameters file to provide a value for this. That is part of why validation is failing.

Reply:
Updated. Added the parameters file.